<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planeposition</class>
 <widget class="QDialog" name="planeposition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>239</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>遇难位置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tubiao.qrc">
    <normaloff>:/新前缀/resource/point_128px_1176459_easyicon.net.ico</normaloff>:/新前缀/resource/point_128px_1176459_easyicon.net.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>纬度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ppweidu1spinBox">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="maximum">
        <number>89</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ppweidu2spinBox">
       <property name="suffix">
        <string>'</string>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ppweidu3spinBox">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>''</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ppweiducomboBox">
       <item>
        <property name="text">
         <string>N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>或为四位小数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ppweidudoubleSpinBox">
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>经度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ppjindu1spinBox">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="maximum">
        <number>179</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ppjindu2spinBox">
       <property name="suffix">
        <string>'</string>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ppjindu3spinBox">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>''</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ppjinducomboBox">
       <item>
        <property name="text">
         <string>E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>或为四位小数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ppjindudoubleSpinBox">
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QPushButton" name="ppokpushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ppcancelpushButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tubiao.qrc"/>
 </resources>
 <connections/>
</ui>
